examples/data_collect/main.cpp

```diff
@@ -110,17 +120,66 @@ static const char kUsageInternal[] =
       --ext_fsync <0|1>  External FSYNC [0: Internal 1: External] [default: 0]
       --ft <frame_type>  Type of frame to be captured [default: raw]
       --wt <warmup>      Warmup Time (in seconds) before data capture [default: 0]
-      --ccb <FILE>       The path to store CCB content      
+      --ccb <FILE>       The path to store CCB content
       --ip <ip>          Camera IP
-      --fw <firmware>    Adsd3500 fw file      
-      --fps <setfps>     Set target FPS value [range: 50 to 200] 
+      --fw <firmware>    Adsd3500 fw file
+      --fps <setfps>     Set target FPS value [range: 50 to 200]
 )";
 
 #ifdef MULTI_THREADED
 void fileWriterTask(const thread_params *const pThreadParams);
 #endif
 
 int main(int argc, char *argv[]) {
+    std::map<std::vector<std::string>, std::string> command_map = {
+        {{"-h", "--help"}, ""},         {{"-f", "--folder"}, "."},
+        {{"-n", "--ncapture"}, "1"},    {{"-m", "--mode"}, "0"},
```

Comment:
This doesn't preserve the exact format of the arguments of data_collect. For instance: only the long option exists for the number of frames (--n). Same for mode.